addon/components/table-column.js

```diff
@@ -102,6 +112,22 @@ export default Component.extend({
     this.set('_registeredParent', table);
   },
 
+  _groupWithPriorRow() {
+    let valueBindingPath = get(this, 'valueBindingPath');
+    let value = getWithDefault(this, `row.${valueBindingPath}`, '');
+
+    // vertical-item > vertical-collection > table-columns
+    // TODO replace with something better
+    let parentView = this.get('parentView.parentView.parentView');
```

Comment:
We have this in 2 places now, so it would probably be better to set a table property. Will add a followup issue for that.